webpack.config.js

```diff
@@ -187,26 +232,26 @@ const webpackConfig = {
 		new CopyWebpackPlugin( [
 			{ from: 'node_modules/flag-icon-css/flags/4x3', to: 'images/flags' },
 		] ),
-		new HappyPack( {
-			loaders: _.compact( [
-				isDevelopment && config.isEnabled( 'webpack/hot-loader' ) && 'react-hot-loader',
-				babelLoader,
-			] ),
-		} ),
-		new webpack.NamedModulesPlugin(),
-		new webpack.NamedChunksPlugin( chunk => {
-			if ( chunk.name ) {
-				return chunk.name;
-			}
-
-			return chunk.modules.map( m => path.relative( m.context, m.request ) ).join( '_' );
-		} ),
-		new NameAllModulesPlugin(),
-		new AssetsPlugin( {
+		//new NameAllModulesPlugin(),
```

Comment:
nix?

Reply:
yeah, breaks the naming that happens via optimize somehow. likely either needs an update or it's not really needed. Need to dig up why we used it in the first place.